app/templates/base.html

```diff
@@ -12,6 +12,11 @@
     <a href="/">Home</a>
     <a href="{{ url_for('publish_new_training_data') }}">New Training data</a>
     <a href="{{ url_for('view_all_data') }}">View data</a>
+    {% if 'auth_key' in session %}
+        <a href="{{ url_for('logout') }}">Logout</a>
```

Comment:
Jak jestem zalogowany, to nie ustawia się logout tylko dalej login. Coś musi być nie tak z tym warunkiem

Reply:
Naprawione, literówka była